cli/app.go

```diff
@@ -361,6 +377,15 @@ var app = &cli.App{
 			Aliases: []string{"q"},
 			Usage:   "suppress warnings",
 		},
+		&cli.StringFlag{
+			Name:  profileFlag,
+			Usage: "Specify a particular profile for the current command",
+		},
+		&cli.BoolFlag{
+			Name:    disableProfilesFlag,
+			Aliases: []string{"disable-profile"},
+			Usage:   "Disable usage of profiles, falling back to default behavior",
```

Comment:
```suggestion
			Usage:   "disable usage of profiles, falling back to default behavior",
```
I'll stop with this one, but _hypothetically_ all the usage texts should probably start with a lower-case letter. This stuff now belongs to you guys (SDK,) though, so up to you at this point.

Reply:
Yeah good callout! Updated all the usage text.